index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" type="text/css" href="css/quiz.css">
+	<title>Ike's Portfolio| Software Projects</title>
+</head>
+<body>
+
+	<nav id="nav_bar">
+		<ul>
+			<li><a href="#">Home</a></li>
+			<li><a href="../projects.html">Projects</a></li>
+			<li><a href="../blog.html">Blog</a></li>
+		</ul>
+	</nav>
+<main>
+	<h1>Ike Nwaelele's Portfolio of Software Projects</h1>
+	<img  id="profile_pic" height="400" width="400" alt="Picture of Ike Nwaelele" src="images/Ike_profile_pic.jpg">
```

Comment:
Try to avoid inline styling when possible. That should happen in the CSS, and ideally you would size the image before including it in the page, because no matter how small you make it with CSS, the browser will still load the image full size before shrinking it, which can be a real drag on the load time for bigger images.